scripts/install.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+# runs setup of base environment and database installs
+
+# create core conda env
+environment_yml = '%s/../%s.yml' % (os.__file__, 'environment')
+os.call('conda env create --name oecophylla -f %s' % environment_yml)
+
+#TODO change this to os.walk
+os.call('find oecophylla -name "*.sh" -execdir bash {} \;')
```

Comment:
`find` does not error out if an individual script returns a non-zero exit status

Reply:
this PR won't be performing install functionality - so this is safe to ignore